dandiapi/api/views/dandiset.py

```diff
@@ -20,10 +21,36 @@
 )
 
 
+class DandisetFilterBackend(filters.OrderingFilter):
+    ordering_fields = ['created', 'name']
+    ordering_description = (
+        'Which field to use when ordering the results. '
+        'Options are created, -created, name, and -name.'
+    )
+
+    def filter_queryset(self, request, queryset, view):
+        orderings = self.get_ordering(request, queryset, view)
+        if orderings:
+            ordering = orderings[0]
+            # ordering can be either 'created' or '-created', so test for both
+            if ordering[-7:] == 'created':
```

Comment:
Just a Python tip: you can use a string's `endswith()` method to avoid having to slice the string: `if ordering.endswith('created'): ...`.